### ISSUE TITLE
High contrast theme: activity bar hover only shown on bottom items

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#143533]
Hover over top items on activity bar - no hover outline
Hover over bottom activity bar items - hover outline

We should make it consistent
![Screenshot 2022-02-21 at 11 43 36](https://user-images.githubusercontent.com/1926584/154939729-528ac7b3-8e12-469b-bc0c-5212c3d07438.png)

**Comment by daviddossett** [comment]
Looks like this is present on the original dark HC theme as well. @misolori confirming this isn't intentional given the different behaviors of the top and bottom activity bar items on click? I can look into fixing it if not. 

<img width="82" alt="CleanShot 2022-02-22 at 14 20 31@2x" src="https://user-images.githubusercontent.com/25163139/155229322-f1ef1e37-55ae-46ef-a4a1-5d0429538b3b.png">

**Comment by misolori** [comment]
I'm not sure why the top and bottom are different but agree that they should all be the same. Given this repros in the dark HC theme it's not a new bug

**Comment by isidorn** [comment]
Yeah it is an old bug. There is probably just some difference in css class name between top and bottom
fyi @sandy081 for activity bar

**Comment by daviddossett** [comment]
I can take care of this—added to next milestone.
